toolkit/templates/report-a-problem.html

```diff
@@ -0,0 +1,22 @@
+<div class="report-a-problem-container">
+  <div class="report-a-problem-inner">
+    <div class="report-a-problem-content">
+      <h2>Help us improve the Digital Marketplace</h2>
+      <form autocomplete="off" action="/feedback" method="post">
+        <input id="uri" name="uri" type="hidden" value="{{ request.url }}">
+
+        <p>Please don’t include personal details or sensitive information about your business or project.</p>
```

Comment:
Does that sufficiently cover government users / civil servants? If so that's fine :)